server/client.go

```diff
@@ -535,24 +728,99 @@ func (c *client) maxPayloadViolation(sz int, max int64) {
 	c.closeConnection()
 }
 
-// Assume the lock is held upon entry.
-func (c *client) sendProto(info []byte, doFlush bool) error {
-	var err error
-	if c.bw != nil && c.nc != nil {
-		deadlineSet := false
-		if doFlush || c.bw.Available() < len(info) {
-			c.nc.SetWriteDeadline(time.Now().Add(c.srv.getOpts().WriteDeadline))
-			deadlineSet = true
+// queueOutbound queues data for client/route connections.
+// Return pending length.
```

Comment:
Not really ;-)

Reply:
Comment on the return or what it does. It queues data for the connection. flushOutbound is what may or may not send to the socket.

Reply:
Was just the comment about returning pending length, which this function does not do (no return value).